js/apps/system/_admin/aardvark/APP/frontend/js/views/graphManagementView.js

```diff
@@ -841,13 +842,45 @@
       var index = str.lastIndexOf(substr);
       return str.substring(0, index);
     },
+    searchKeyDown: function (event) {
+      if (
+        event && event.originalEvent && (
+          (
+            event.originalEvent.key &&
+            (
+              event.originalEvent.key === 'Control' || 
+              event.originalEvent.key === 'Alt' || 
+              event.originalEvent.key === 'Shift'
+            )
+          ) || 
+          event.originalEvent.ctrlKey || 
+          event.originalEvent.altKey ||
+          event.originalEvent.metaKey
```

Comment:
Should we consider moving this logic to a common place? Will become a little more readable I suppose..